api/views/phones.py

```diff
@@ -439,17 +439,20 @@ def inbound_sms(request):
         )
 
     _check_disabled(relay_number, "texts")
-
+    _check_remaining(relay_number, "texts")
     inbound_contact = _get_inbound_contact(relay_number, inbound_from)
     if inbound_contact:
-        _check_and_update_contact(inbound_contact, "texts")
+        _check_and_update_contact(inbound_contact, "texts", relay_number)
 
     client = twilio_client()
     client.messages.create(
         from_=relay_number.number,
         body=f"[Relay 📲 {inbound_from}] {inbound_body}",
         to=real_phone.number,
     )
+    relay_number.remaining_texts -= 1
```

Comment:
Can you update an incoming SMS test to check the value of `relay_number.remaining_texts`?